### PR TITLE
✨ Use +kubebuilder:skip to skip package

### PR DESCRIPTION
**Pull request by tamalsaha**
Signed-off-by: Tamal Saha <tamal@appscode.com>

Fix https://github.com/kubernetes-sigs/controller-tools/issues/286

### CODE REVIEW

File: pkg/crd/markers/package.go [hunk 1]
```diff
@@ -33,5 +33,8 @@ func init() {
 
 		must(markers.MakeDefinition("kubebuilder:validation:Required", markers.DescribesPackage, struct{}{})).
 			WithHelp(markers.SimpleHelp("CRD validation", "specifies that all fields in this package are required by default.")),
+
+		must(markers.MakeDefinition("kubebuilder:skip", markers.DescribesPackage, false)).
```
Comment (DirectXMan12) [inline pkg/crd/markers/package.go]: I think we can probably just make this an empty marker, so you just do

```go
// +kubebuilder:skip
package internal_version
```

File: pkg/crd/markers/package.go [hunk 2]
```diff
@@ -33,5 +33,8 @@ func init() {
...
+			WithHelp(markers.SimpleHelp("CRD", "skip this package.")),
```
Comment (DirectXMan12) [inline pkg/crd/markers/package.go]: nit: probably clearer to say "don't consider this package as an API version"

File: pkg/crd/testdata/cronjob_types.go [hunk 3]
```diff
@@ -17,6 +17,7 @@ limitations under the License.
 
 // +groupName=testdata.kubebuilder.io
 // +versionName=v1
+// +kubebuilder:skip=true
```
Comment (DirectXMan12) [inline pkg/crd/testdata/cronjob_types.go]: we shouldn't skip the main test package -- add a new package for this

**Comment by tamalsaha** [comment]
@DirectXMan12 , I think I have addressed your comments. PTAL.

**Comment by DirectXMan12** [comment]
/lgtm
/approve

thanks!

**Pull request merged**